main.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module Enumerable #:nodoc:
+  def my_each
+    i = 0
+    while i < size
+      yield(self[i])
+      i += 1
+    end
+    self
+  end
```

Comment:
- [ ] my_each should return an **Enumerator** if no block is given. 
- Hint: Use `if block_given?` at the beginning of the method :wink: